poetry.lock

```diff
@@ -0,0 +1,18 @@
+# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```

Comment:
What's the consequence of this file if, for example, you were to step away and I didn't update this with future updates? Will this file need to change with releases, or is it expected to remain mostly static? Would these hashes be universal, or system specific to you?

Reply:
According to the poetry documentation, the hashes are based on the dependencies. It'd automatically handled by poetry